version/src/main/java/org/jboss/as/version/ProductConfig.java

```diff
@@ -81,10 +84,18 @@ private ProductConfig(ModuleLoader loader, ProductConfProps productConfProps, Ma
                         minStability = Stability.fromString(minStabilityValue);
                     }
                 }
+                Path bannerPath = getBannerFile(home);
+                if (bannerPath != null && Files.exists(bannerPath)) {
+                    try {
+                        projectBanner = System.lineSeparator() + Files.readString(bannerPath);
+                    } catch (Exception e) {
+                        // Don't care
+                    }
+                }
             }
-
             setSystemProperties(productConfProps.miscProperties, providedProperties);
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
If we want the trace, I would say we have to use a logger. Printing the trace on the standard error doesn't look correct to me.